src/core/src/package_managers/PackageManager.py

```diff
@@ -316,6 +320,42 @@ def set_package_manager_setting(self, setting_key, setting_value=""):
         self.package_manager_settings[setting_key] = setting_value
     # endregion
 
+    # region auto OS updates
+    def disable_auto_os_update(self):
+        pass
+
+    def log_default_auto_os_updates(self):
+        """ Records the default system settings for auto OS updates within patch extension artifacts for future reference.
+        We only log the default setting a VM comes with, any subsequent updates will not be recorded"""
+        if not self.default_auto_os_update_log_exists():
+            current_auto_os_update_settings = self.get_current_auto_os_update_settings()
+            if current_auto_os_update_settings is not None:
+                self.composite_logger.log_debug("Logging default system settings for auto OS updates. [Settings={0}] [Log file path= {1}]"
+                                                .format(str(current_auto_os_update_settings), self.default_auto_os_update_log_path))
+                self.env_layer.file_system.write_with_retry(self.default_auto_os_update_log_path, '[{0}]'.format(current_auto_os_update_settings), mode='w+')
+
+    def default_auto_os_update_log_exists(self):
```

Comment:
Scan through the code and comments for standard terminology discussed elsewhere in the PR. Can discuss in call.